<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIncendios</class>
 <widget class="QMainWindow" name="frmIncendios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultas de Incendios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tblIncendiosProvincias">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>791</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txtCantidad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cantidad de incendios por provincia</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtProvincia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Provincia:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbProvincias">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkGraficar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Graficar todas las cantidades</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>791</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
